gasometer/src/costs.rs

```diff
@@ -39,18 +39,10 @@ pub fn sstore_refund(original: H256, current: H256, new: H256, config: &Config)
 				}
 
 				if original == new {
-					let (gas_sstore_reset, gas_sload) = if config.increase_state_access_gas {
-						(
-							config.gas_sstore_reset - config.gas_sload_cold,
-							config.gas_storage_read_warm,
-						)
-					} else {
-						(config.gas_sstore_reset, config.gas_sload)
-					};
```

Comment:
I decided that doing this arithmetic at runtime was sub-optimal, so I pushed it into constructing the config itself with an addition `const fn` that takes some arguments. Computing these derived parameters at the level of constructing the config also makes the change to `refund_sstore_clears` in EIP-3529 cleaner.